workflow-templates/deploy-application.yml

```
@@ -0,0 +1,159 @@
+# Workflow to deploy to an AWS environment with maven, terraform, build-image
+# This workflow requires the maven wrapper to be in the root of the project
+
+name: 'Deploy to AWS'
```

```suggestion
name: Deploy to AWS
```

Bij de andere staat er ook geen ' rounds